tailcfg/tailcfg.go

```diff
@@ -2257,6 +2257,46 @@ type SSHRecordingAttempt struct {
 	FailureMessage string
 }
 
+// EnableFeatureRequest is a request sent to "/machine/f/{feature}/enable"
+// to enable a feature, such as Funnel, for the node's tailnet.
+//
+// This request itself DOES NOT modify tailnet configuration to enable the
+// feature, but returns information that can be presented to the user about
+// where/how to enable the feature. See EnableFeatureResponse for response
+// structure.
+type EnableFeatureRequest struct {
+	// NodeKey is the client's current node key.
+	NodeKey key.NodePublic `json:",omitempty"`
+}
+
+// EnableFeatureResponse is the response to an EnableFeatureRequest.
+type EnableFeatureResponse struct {
+	// Complete is true when the feature is already enabled.
+	Complete bool `json:",omitempty"`
+
+	// Text holds lines to display in the CLI with information
+	// about the feature and how to enable it.
+	//
+	// Lines are separated by newline characters. No trailing
+	// newline.
```

Comment:
last sentence isn't a full sentence. But maybe leave more wiggle worm. "The final newline may be omitted." ?